vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -49,33 +49,42 @@ void Notifications::Initialize(InitializeCallback callback) {
 bool Notifications::Get(const std::string& id, NotificationInfo* info) const {
   DCHECK(is_initialized_);
 
-  auto notification = notifications_.find(id);
-  if (notification == notifications_.end()) {
+  auto iter = std::find_if(notifications_.begin(), notifications_.end(),
+      [&id](const auto & v){return v.id == id;});
```

Comment:
Can we please change v to notification.

Can we remove the space between auto and & as this will fail linter

Reply:
done